### DISCUSSION TITLE
DDT4ALL and PyRen on SteamDeck

### DISCUSSION THREAD

**Discussion by DavidIgor** [cedricp/ddt4all#1046] [Q&A]
hi @cedricp I have managed to install ddt4all and database on my SteamDeck (all works)! However I have trouble with pyren ... where can I get latest version? (as version I have is from 2018!?)
thanks

**Comment by AndyIreland** [comment]
https://gitlab.com/py_ren/pyren/-/tree/pyren3/pyren3

**Reply by DavidIgor** [reply]
@AndyIreland thanks for this!  does it have database? (Ill test tomorrow)

**Reply by DavidIgor** [reply]
Oh Ill use my ddt database

**Comment by AndyIreland** [comment]
> Oh Ill use my ddt database

yes that's right you can use your own ecu.zip

**Reply by DavidIgor** [reply]
![image](https://github.com/cedricp/ddt4all/assets/122531259/81b86eff-d490-4d1f-bd97-6c92a6ee4753)

**Comment by AndyIreland** [comment]
sorry for confusion I thought you were talking using PyREN on Windows laptop.

**Comment by KarelSvo** [comment]
@DavidIgor 
Email sent
